### PR TITLE
ZER-59 Add login page for Admin user

### PR DESCRIPTION
**Pull request by haliapats**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by maksymst — CHANGES_REQUESTED** [review]
1. Leave only templates required for sign in
2. Use https://github.com/slim-template/slim for templates. It is already in the master branch

File: db/migrate/20210513165126_devise_create_admins.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      ## Confirmable
+      # t.string   :confirmation_token
+      # t.datetime :confirmed_at
+      # t.datetime :confirmation_sent_at
+      # t.string   :unconfirmed_email # Only if using reconfirmable
+
+      ## Lockable
+      # t.integer  :failed_attempts, default: 0, null: false # Only if lock strategy is :failed_attempts
+      # t.string   :unlock_token # Only if unlock strategy is :email or :both
+      # t.datetime :locked_at
```
Comment (maksymst) [inline db/migrate/20210513165126_devise_create_admins.rb:33]: you have these features enabled in Admin model

**Review by maksymst — CHANGES_REQUESTED** [review]

File: app/controllers/admins/sessions_controller.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class Admins::SessionsController < Devise::SessionsController
+  include Accessible
+  skip_before_action :check_user, only: :destroy
```
Comment (maksymst) [inline app/controllers/admins/sessions_controller.rb:5]: Why do you need this?

File: app/views/admins/confirmations/new.html.slim [hunk 3]
```diff
@@ -0,0 +1,11 @@
+h2
+  | Resend confirmation instructions
```
Comment (maksymst) [inline app/views/admins/confirmations/new.html.slim:2]: All texts in templates must be localized

**Review by maksymst — CHANGES_REQUESTED** [review]

File: app/controllers/admins/sessions_controller.rb [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  layout 'admin'
+  before_action :check_user
+
+  protected
+  def check_user
+    if current_admin
+      flash.clear
+      redirect_to('/admin') and return
+    end
+  end
```
Comment (maksymst) [inline app/controllers/admins/sessions_controller.rb:13]: https://stackoverflow.com/questions/31943125/how-to-correctly-use-guard-clause-in-ruby

**Review by maksymst — APPROVED** [review]

**Review by IvanShvets42 — APPROVED** [review]

**Pull request merged**